<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QMainWindow" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="1" column="0">
     <widget class="QTableWidget" name="tableWidget">
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>0</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="pauseButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Pause</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>37</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLaunch_Disc_Game"/>
    <addaction name="actionOpen_ELF"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionSystem"/>
    <addaction name="actionFilesystem"/>
    <addaction name="actionLLE"/>
    <addaction name="separator"/>
    <addaction name="actionPPU"/>
    <addaction name="actionSPU"/>
    <addaction name="actionRSX"/>
    <addaction name="actionAudio"/>
    <addaction name="separator"/>
    <addaction name="actionDebug"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionPPU_Debugger"/>
    <addaction name="actionThread_Debugger"/>
    <addaction name="separator"/>
    <addaction name="actionReplay_RSX_Capture"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_ChonkyStation3"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuDebug"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_ELF">
   <property name="text">
    <string>Launch ELF</string>
   </property>
  </action>
  <action name="actionSystem">
   <property name="text">
    <string>System</string>
   </property>
  </action>
  <action name="actionLLE">
   <property name="text">
    <string>LLE</string>
   </property>
  </action>
  <action name="actionDebug">
   <property name="text">
    <string>Debug</string>
   </property>
  </action>
  <action name="actionFilesystem">
   <property name="text">
    <string>Filesystem</string>
   </property>
  </action>
  <action name="actionPPU">
   <property name="text">
    <string>PPU</string>
   </property>
  </action>
  <action name="actionSPU">
   <property name="text">
    <string>SPU</string>
   </property>
  </action>
  <action name="actionRSX">
   <property name="text">
    <string>RSX</string>
   </property>
  </action>
  <action name="actionReplay_RSX_Capture">
   <property name="text">
    <string>Replay RSX Capture</string>
   </property>
  </action>
  <action name="actionLaunch_Disc_Game">
   <property name="text">
    <string>Launch Disc Game</string>
   </property>
  </action>
  <action name="actionThread_Debugger">
   <property name="text">
    <string>Thread Debugger</string>
   </property>
  </action>
  <action name="actionPPU_Debugger">
   <property name="text">
    <string>PPU Debugger</string>
   </property>
  </action>
  <action name="actionAudio">
   <property name="text">
    <string>Audio</string>
   </property>
  </action>
  <action name="actionAbout_ChonkyStation3">
   <property name="text">
    <string>About ChonkyStation3</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
